web/src/routes/_layout.svelte

```diff
@@ -8,7 +8,7 @@
 <style>
 	main {
 		position: relative;
-		max-width: 56em;
+		max-width: 80vw;
```

Comment:
I'm a fan of the viewport units. But later I may change this over to use some more content-based units like ch or just pain old rem and Ems.